tests/models/align/test_modeling_align.py

```diff
@@ -482,33 +482,28 @@ def test_model_common_attributes(self):
         pass
 
     # override as the `temperature` parameter initilization is different for ALIGN
-    def test_initialization(self):
```

Comment:
[amyeroberts](https://github.com/amyeroberts) Pending
I'd suggest looking at the changes in [test_modeling_common.py](https://github.com/huggingface/transformers/pull/30451/files#diff-a6ebb0bccfe159ca855e9d9b5d6fbbeb62fced0c0296de0993ba39ef5c862cf0) as this will provide more context for the changes in the other model's tests :)